CoinDock-api/app/Console/Kernel.php

```diff
@@ -7,15 +7,12 @@
 
 class Kernel extends ConsoleKernel
 {
-    /**
-     * Define the application's command schedule.
-     *
-     * @param  \Illuminate\Console\Scheduling\Schedule  $schedule
-     * @return void
-     */
+
     protected function schedule(Schedule $schedule)
     {
         $schedule->command('passport:purge')->hourly();
+        $schedule->command('wallet:handle_balance')->everyFiveMinutes();
```

Comment:
can we add runInBackground() and withoutOverlapping() to these commands

Reply:
done